pkg/common/aws/utils.go

```diff
@@ -18,34 +19,36 @@ import (
 type ccsAwsSession struct {
 	session *session.Session
 	iam     *iam.IAM
+	ec2     *ec2.EC2
 	once    sync.Once
 }
 
 // CCSAWSSession is the global AWS session for interacting with AWS.
 var CcsAwsSession ccsAwsSession
 var ccsKeys *iam.CreateAccessKeyOutput
 
-//GetSession returns a new AWS session with the first AWS account in the config file. The session is cached for the rest of the program.
-func (CcsAwsSession *ccsAwsSession) getIamClient() (*session.Session, *iam.IAM) {
+// GetSession returns a new AWS session with the first AWS account in the config file. The session is cached for the rest of the program.
+func (CcsAwsSession *ccsAwsSession) getClient() (*session.Session, *iam.IAM, *ec2.EC2) {
```

Comment:
can we rename this to `getClients()` and update the comment since it is doing more than the comment above suggests

Reply:
Pushed a new PR. 
How about GetAWSSessions? Since it's just wrapping session.NewSession

Reply:
SGTM! Thanks for the update